apps/ui/src/composables/useEditor.ts

```diff
@@ -128,6 +128,7 @@ export function useEditor() {
       discussion: '',
       type,
       choices,
+      privacy: '',
```

Comment:
with out `privacy` we need to pass [this type](https://github.com/snapshot-labs/sx-monorepo/blob/0f199c1ae6ff5731c09727645472eb8fe3569f1b/packages/sx.js/src/clients/offchain/ethereum-sig/types.ts#L77C5-L78C1) on a condition 🙈 will add a layer of confusion

maybe we should just pass `shutter` by default if space uses shutter?

Reply:
> maybe we should just pass shutter by default if space uses shutter?

In that case we need to handle it on sequencer, there is also a case where `privacy` is undefined ( if some integrator is using a old version of snapshot)

Reply:
> maybe we should just pass shutter by default if space uses shutter?

That was was I was thinking, that `privacy` should reflect was is actually used instead of empty string being there where space enforces shutter. If space uses `shutter` we can't pass `shutter` there?

Reply:
Now we pass `shutter` to proposal if space is using `shutter` https://github.com/snapshot-labs/sx-monorepo/pull/954/commits/bd5448071a925fbb864917f09917b217c3eb367d
and on sequencer https://github.com/snapshot-labs/snapshot-sequencer/pull/475